### ISSUE TITLE
How do I start developing code with you guys?

### ISSUE THREAD

**Issue by dirktheeng** [grbl/grbl#24]
This isn't an issue, but I don't know how else to post on this site.  I am using the edge branch code to develop an open source controller specifically meant for CNC laser users.  I built a buildlog 2.X laser (www.buildlog.net) and am using this for control as it has almost everything I need.  That said, I am adding features and configuration options that are needed for lasers and that would generally make the code easier to use with a variety of stepper driver configurations.  The first thing I did was add a configuration option that sets the steper enable pin to Enable=high, sleep; Enable=high, no sleep; Enable=low, sleep; Enable=low, no sleep.  I also wrote two small functions that set the enable on/off state as well as the motor trigger interrupt based on those options.  This was necessary because certain drivers need a high or low to enable the steppers (case in point: the GRBL sheild needs a low while the buildlog laser needs a high).  It is also not always advantageous to put the motors to sleep inbetween gcode sets as somebody can move them and the controller wouldn't know about it.

My next task is to add an M-code to put the motors to sleep or wake them up.  Laser users will want that as there are times when we want to roughly position the laser by hand and then lock the motors, set zero, and start.

I want to move the code from a 328p system to the Mega as I don't have enough IO for everythign I want to do.  I also want to develop this into a 4 and 5 axis system eventually.

My ultimate goal is to feed this with another arduino reading code from a SD card and interfacing with my computer

Basically, I want to put this code out there for others to get and use as they see fit, but I don't know how.
